pkg/encryption/encryption.go

```diff
@@ -21,13 +21,19 @@ import (
 	"encoding/binary"
 	"fmt"
 	"hash"
+
+	"github.com/ethersphere/bee/pkg/swarm"
 )
 
 const (
 	KeyLength     = 32
 	ReferenceSize = 64
 )
 
+const (
+	EncryptionBranches = swarm.Branches / 2
```

Comment:
this constant better in `swarm` package no?

Reply:
That was my initial idea, but then I moved it here. Will move it back.